eng/pipelines/pipeline-generation.yml

```diff
@@ -26,6 +26,8 @@ parameters:
     default: 76
   - name: APIReview_AutoCreate_Configurations
     default: 93
+  - name: AzureSdkArtifacts_SAS_tokens
```

Comment:
Before we take this change lets see how we handle the public ci jobs as some solutions might not need this change at all.